string message
int64 value
---
bool result
string message
